components/text-utilities/text-utilities.css

```diff
@@ -0,0 +1,23 @@
+.txt-small{
+    font-size: small;
+}
+
+.txt-gray{
+    color: var(--darkgrey);
+   font-weight: bold;
+}
+.txt-align{
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
Since you are aligning using flex, maybe try to use a better classname like `txt-center-flex`

Reply:
Okay will make that change. Thanks.